admin-dev/themes/new-theme/package.json

```diff
@@ -17,16 +17,18 @@
   "license": "OSL-3.0",
   "dependencies": {
     "babel-runtime": "^6.26.0",
+    "bootstrap-colorpicker": "^3.2.0",
     "bootstrap-slider": "^6.0.16",
     "bootstrap-tokenfield": "^0.12.0",
     "bourbon": "^4.2.6",
     "compass": "^0.1.1",
+    "decimal.js": "^10.2.1",
```

Comment:
Use [big](http://mikemcl.github.io/big.js/), not decimal

[read why](https://github.com/MikeMcl/big.js/wiki)